core/src/main/clojure/xtdb/pgwire.clj

```diff
@@ -1878,6 +1892,10 @@
           (try
             (let [conn-socket (.accept accept-socket)]
               (.setTcpNoDelay conn-socket true)
+              (.setSendBufferSize conn-socket socket-buffer-size)
+              (.setReceiveBufferSize conn-socket socket-buffer-size)
+              (log/info "Accepted connection" {:send-buffer-size (.getSendBufferSize conn-socket),
```

Comment:
we usually recommend that users run XT at info, so I'd guess this is debug?

Reply:
Good catch. That is a leftover from me as the OS sometimes can set them as well.

Reply:
Ah, while removing it, I actually saw that the 1kb is not exactly honoured:
```
Accepted connection {:send-buffer-size 2304, :receive-buffer-size 1152}
```